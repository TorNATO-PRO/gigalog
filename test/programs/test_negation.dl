negate(X, Y) :- Y = -X.

negate(5, -5).
negate(-8, 8).
negate(0, 0).
