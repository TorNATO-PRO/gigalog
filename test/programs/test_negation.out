negate(-8,8)
negate(0,0)
negate(5,-5)
